Phonebook.Source.MockPeopleBackend/package-lock.json

```diff
@@ -4,26 +4,10 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```

Comment:
please revert the changes in this file. They are not relevant for the kubernetes deployment